BlazarUI/app/scripts/components/repo-build/RepoBuildDetail.jsx

```diff
@@ -104,14 +88,23 @@ class RepoBuildBuildDetail extends Component {
   
     if (contains(FINAL_BUILD_STATES, build.state)) {
       buildDetail.endtime = timestampFormatted(build.endTimestamp)
+
+      if (build.state === BuildStates.CANCELLED) {
```

Comment:
looks good. we could probably simplify this just a little more by assigning the `after` or `in` as a variable, and then only need to define `buildDetail.duration` once.

eg something like
```
const conjunction = build.state === BuildStates.CANCELLED ? 'after' : 'in';

buildDetail.duration = (
  <span>{conjunction} {build.duration}</span>
)
```